README.md

```diff
@@ -2,32 +2,24 @@
 This module contains plugins and some example migrations to import data from a Fedora 3 Islandora instance
 into an Islandora CLAW instance.
 
-This is a base setup, it requires adjustments to the default Repository Object and configuration changes
-for your setup.
+This is a base setup, it requires configuration changes for your setup.
 
-## Required changes
-The default Repository Object provided with Islandora CLAW requires one additional field to allow for
-these migrations (or you can comment out these field migrations).
+## Installation and configuration
+To use this migration:
 
-1. A large text field called `field_mods_text`, this will store the MODS datastream from the source object.
+1. from within your Drupal 8 base directory (e.g. `/var/www/html/drupal`), run `composer require islandora/migrate_7x_claw`
+1. edit the .yaml files as indicated in the next section (it is important you edit these files before installing the module)
 
-This is defined in the `config/install/migrate_plus.migration.islandora_basic_image.yml` and
-can be commented out there.
+## Editing the .yaml configuration files
 
-## Example usage
-To use this migration, clone this repo into your Drupal 8 instance `modules/contrib` directory. 
-
-DO NOT INSTALL THE MODULE YET!!!
-
-You will need to edit the 3 `migrate_plus.migration.islandora_basic_image*` files in the `config/install` directory.
+You will need to edit the 3 `migrate_plus.migration.islandora_basic_image*` files in the `migrate_7x_claw/modules/islandora_migrate_7x_claw_feature/config/install` directory.
 
 At a minimum you'll need to set:
 1. `solr_base_url: http://10.0.2.2:9080/solr` to your Solr instance
-1. `fedora_base_url: &fedora_base_url http://10.0.2.2:9080/fedora` to your Fedora, please leave the `&fedora_base_url`
-this is a placeholder and saves re-typing this value in other locations.
+1. `fedora_base_url: &fedora_base_url http://10.0.2.2:9080/fedora` to your Fedora.
```

Comment:
The `&fedora_base_url` was the anchor that could be called in other locations to repeat a Yaml node. If we are removing this explanation, then we should remove the `&fedora_base_url` as well and leave it up to the end user. These anchors seem to have been removed from the actual configuration files anyways.

Reply:
@whikloj  do these anchors work across yaml files or just within a single file?

Reply:
Pretty sure it's within the context of a single file.  We'd have to dig into Drupal migrate to see if there's a way we can share variables across migrations.

Reply:
Well the `&fedora_base_url` no longer exist in the configuration files so we should probably remove them from the documentation. If we decide to add them back in we can update the docs then.